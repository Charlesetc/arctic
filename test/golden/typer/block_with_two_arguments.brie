define main [ : x y [ x ] ]
