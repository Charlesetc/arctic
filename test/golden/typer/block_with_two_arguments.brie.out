{[2, 0, 3]=>Unknown(), [4]=>Function_literal(takes=1, returns=3), [5, 6]=>Function_literal(takes=0, returns=4)}